src/audio/VoiceRecording.ts

```diff
@@ -135,15 +156,17 @@ export class VoiceRecording extends EventEmitter implements IDestroyable {
                 this.recorderProcessor.addEventListener("audioprocess", this.onAudioProcess);
             }
 
+            const hqRecording = this.shouldRecordInHighQuality();
             this.recorder = new Recorder({
                 encoderPath, // magic from webpack
                 encoderSampleRate: SAMPLE_RATE,
-                encoderApplication: 2048, // voice (default is "audio")
+                encoderApplication: hqRecording ? higQualityRecorderOptions.encoderApplication
+                    : voiceRecorderOptions.encoderApplication,
                 streamPages: true, // this speeds up the encoding process by using CPU over time
                 encoderFrameSize: 20, // ms, arbitrary frame size we send to the encoder
                 numberOfChannels: CHANNELS,
                 sourceNode: this.recorderSource,
-                encoderBitRate: BITRATE,
+                encoderBitRate: hqRecording ? higQualityRecorderOptions.bitrate : voiceRecorderOptions.bitrate,
```

Comment:
Again minor and this is probably just personal preference (will also caveat with the fact that I'm not too familiar with this repo yet). 

What would you think of something like the following?
```
const recorderOptions = this.shouldRecordInHighQuality() ? highQualityRecorderOptions : voiceRecorderOptions;
const {encoderApplication, encoderBitRate} = recorderOptions;
this.recorder = new Recorder({
  ...
  encoderApplication,
  encoderBitRate,
  ...
})
```

 I think it would slightly reduce the chance of somebody accidentally changing one of these ternaries (but missing the other one) in the future.

Reply:
It looks much better. Thank you.